src/urllib3/contrib/ssl.py

```diff
@@ -0,0 +1,199 @@
+import ssl
+import socket
+import io
+
+
+class SSLTransportError(Exception):
```

Comment:
Why do we have a separate Exception type over reraising the `ssl.SSLError` within `_ssl_io_loop`?

Reply:
No good reason to be honest, it's been removed.